sdk/pinocchio/Cargo.toml

```diff
@@ -19,3 +19,6 @@ unexpected_cfgs = { level = "warn", check-cfg = [
 
 [features]
 std = []
+
+[build-dependencies]
+rustc_version = "0.4"
```

Comment:
It does not seem that this PR is changing anything in the `build.rs`.
Is this dependency still necessary?